src/ValueFormatters/QuantityFormatter.php

```diff
@@ -50,15 +61,17 @@ class QuantityFormatter extends ValueFormatterBase {
 	/**
 	 * @param DecimalFormatter|null $decimalFormatter
 	 * @param FormatterOptions|null $options
+	 * @param QuantityUnitFormatter $unitFormatter
 	 */
-	// FIXME: In all other ValueFormatters the FormatterOption parameter is the first one.
-	public function __construct( DecimalFormatter $decimalFormatter = null, FormatterOptions $options = null ) {
+	public function __construct( QuantityUnitFormatter $unitFormatter, DecimalFormatter $decimalFormatter = null, FormatterOptions $options = null ) {
```

Comment:
The order of things confuses me. Why is the unit first? But last down in the constructor code?